client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java

```diff
@@ -199,6 +207,85 @@ public void onFailure(Exception e) {
         }
     }
 
+    @SuppressWarnings({ "unused", "unchecked" })
+    public void testSearchRequest_Aggregations() throws IOException {
+        RestHighLevelClient client = highLevelClient();
+        {
+            BulkRequest request = new BulkRequest();
+            request.add(new IndexRequest("posts", "doc", "1")
+                    .source(XContentType.JSON, "company", "Elastic", "age", 20));
+            request.add(new IndexRequest("posts", "doc", "2")
+                    .source(XContentType.JSON, "company", "Elastic", "age", 30));
+            request.add(new IndexRequest("posts", "doc", "3")
+                    .source(XContentType.JSON, "company", "Elastic", "age", 40));
+            request.setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE);
+            BulkResponse bulkResponse = client.bulk(request);
+            assertSame(bulkResponse.status(), RestStatus.OK);
+            assertFalse(bulkResponse.hasFailures());
+        }
+        {
+            SearchRequest searchRequest = new SearchRequest();
+            // tag::search-request-aggregations
+            SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
+            TermsAggregationBuilder aggregation = AggregationBuilders.terms("by_company").field("company.keyword")
+                    .subAggregation(AggregationBuilders.avg("average_age").field("age"));
```

Comment:
Good point, I will do that. 
I wonder what we do in these cases once we start having auto-formatting (there was a discussion around this at some point).